client/js/app/stores/ExplorerStore.js

```diff
@@ -283,6 +283,10 @@ function _wrapGroupBy(group_by) {
   return group_by;
 }
 
+function _coerceToNumber(str) {
+  return str && typeof(str) === 'string' ? Number(str) : str;
```

Comment:
Yup, looks like we need to account for both `null` and empty strings. I'd like to move forward by removing `_coerceToNumber` and using `parseFloat()` directly on lines 306 & 320: 

306: `newAttrs.query.percentile = parseFloat(newAttrs.query.percentile);`
320: `newModel.query.percentile = parseFloat(newModel.query.percentile);`

If the percentile is `NaN`, the query will error and the chart will continue to hang in a loading state (consistent with current behavior).